java/arcs/android/storage/service/BindingContext.kt

```diff
@@ -125,6 +128,7 @@ class BindingContext(
                         }
                         resultCallback.onResult(null)
                     } catch (e: Throwable) {
+                        log.warning(e) { "during idle" }
```

Comment:
maybe use the same strings as the CrdtExceptions instead "Exception occurred while awaiting idle" etc so when looking at logs it is clear it is the same exception

Reply:
Done.